### DISCUSSION TITLE
Support `git subtree`

### DISCUSSION THREAD

**Discussion by debuglevel** [community/community#19757] [Codespaces]
`git subtree` does not work on current the default image:
```bash
$ git subtree
git: 'subtree' is not a git command. See 'git --help'.
```

It is actually available at `/usr/lib/git-core/git-subtree` and works if you call it by its full path.

```bash
$ find / -name '*subtree*' 2> /dev/null
/usr/lib/git-core/git-merge-subtree
/usr/lib/git-core/git-subtree
/usr/share/man/man1/git-subtree.1.gz
/usr/share/doc/git/contrib/subtree
/usr/share/doc/git/contrib/subtree/t/t7900-subtree.sh
/usr/share/doc/git/contrib/subtree/git-subtree.sh
/usr/share/doc/git/contrib/subtree/git-subtree
/usr/share/doc/git/contrib/subtree/git-subtree.txt
/usr/local/libexec/git-core/git-merge-subtree
```

**Comment by sardorb3k** [comment]
https://cogumbreiro.blogspot.com/2013/05/how-to-install-git-subtree-in-ubuntu.html

**Reply by debuglevel** [reply]
The command `sudo ln -s /usr/share/doc/git/contrib/subtree/git-subtree.sh /usr/lib/git-core/git-subtree` suggests that it's only necessary to link `/usr/share/doc/git/contrib/subtree/git-subtree.sh` to `/usr/lib/git-core/git-subtree`. As shown above, this is already available.

**Comment by kallayj** [comment]
Running `git --exec-path` returns `/usr/local/libexec/git-core`. `git-subtree` is missing from that directory. `/usr/local/libexec` contains _only_ the `git-core` directory, so special care was taken to run git out of this directory (presumably to ensure git commands are auto-magically integrated with Github). I'm curious if subtree is intentionally or accidentally excluded.
